runtimes/eden/src/pallets_util.rs

```diff
@@ -232,18 +250,25 @@ parameter_types! {
 	pub const MaxSubAccounts: u32 = 100;
 	pub const MaxAdditionalFields: u32 = 100;
 	pub const MaxRegistrars: u32 = 20;
+	pub const ByteDeposit: Balance = constants::deposit(0, 1); // TODO give this a reasonable value
```

Comment:
@aliXsed  any sugestions?

Reply:
I think your current choice is good. Because it would be consistent with the preimage pallet. Currently deposit(0, 1) returns 600 MicroNodls. The only non consistent setting is the one we have used for NFT metadata per byte charge with is 10 MilliNodls. I would suggest you change the function `deposit` to return 10 MilliNodls for one byte and use it for NFT config as well.

Reply:
```suggestion
	pub const ByteDeposit: Balance = constants::deposit(0, 1);
```